crates/rolldown/tests/integration_rolldown.rs

```diff
@@ -61,5 +62,15 @@ async fn filename_with_hash() {
 
     snapshot_outputs.push(snapshot_output);
   }
+  let snapshot_path = workspace::crate_dir("rolldown")
+    .join("tests/snapshots/integration_rolldown__filename_with_hash.snap");
+  let before_snapshot = std::fs::read_to_string(&snapshot_path).unwrap();
+
   insta::assert_snapshot!(snapshot_outputs.join("\n"));
+
+  let in_github_action = std::env::var("GITHUB_ACTIONS").is_ok();
+  let after_snapshot = std::fs::read_to_string(&snapshot_path).unwrap();
+  if (before_snapshot != after_snapshot && in_github_action) {
+    std::process::exit(-1);
```

Comment:
Should  this be done by insta?

https://github.com/rolldown/rolldown/blob/dde9b33b409bfc909d089769d2817f178e00f30f/.github/workflows/reusable-cargo-test.yml#L36-L37

Reply:
you are right.

Reply:
https://github.com/rolldown/rolldown/actions/runs/10431525166/job/28891291395?pr=2006, but actually it did not work

Reply:
![image](https://github.com/user-attachments/assets/87dc0714-5e9f-4f64-ad73-8fbd27e232cc)

Reply:
I agree with you that setting the env is correct way to fix this issue.